tv_apps/smart-home/js/card_picker.js

```diff
@@ -114,8 +120,21 @@
         }
 
         var appButton = CardUtil.createCardButton(card, true);
-        that.gridView.appendChild(appButton);
-        that.appButtons.push(appButton);
+        var key = String(defaultOrder[card.group] || 5);
+        if (!(card instanceof Deck)) {
+          var lang = document.documentElement.lang;
+          var name = that._cardManager.resolveCardName(card, lang);
+          if (name.raw) {
+            key += name.raw;
+          } else if (name.args && name.args.number !== undefined) {
```

Comment:
Please describe it explicitly the purpose of this if-condition that it's for TV cards.